sys/kern/src/syscalls.rs

```diff
@@ -69,6 +75,23 @@ pub unsafe extern "C" fn syscall_entry(nr: u32, task: *mut Task) {
     };
 
     with_task_table(|tasks| {
+        // On certain architectures we risk receiving "phantom SVCs" assigned to
+        // tasks that are not, in fact, making system calls. As of this writing,
```

Comment:
This module is attempting not to be ARM specific, so I'm not sure how I would do that. I could tell people to go read `arch/arm_m.rs` maybe?

Reply:
Yeah, that's reasonable; I think it might be reasonable to say "on certain ARM architectures (see ...)" or something here. I'm just always leery of comments that say "$SPOOKY_THING *might* happen, but I won't tell you when and where it might happen"...